balancer/grpclb/grpc_lb_v1/load_balancer_grpc.pb.go

```diff
@@ -154,3 +154,11 @@ var LoadBalancer_ServiceDesc = grpc.ServiceDesc{
 	},
 	Metadata: "grpc/lb/v1/load_balancer.proto",
 }
+
```

Comment:
On line 57 (inside the `BalanceLoad` method on the client), can we make this use the generated const instead of hard-coding the string again?

Reply:
Just pushed some changes where the new symbols replace any hard coded strings, so it only appears in the const declaration